coalib/coala_delete_orig.py

```diff
@@ -26,20 +22,21 @@ def main(log_printer=None, section: Section=None):
 
     not_deleted = 0
     for ofile in orig_files:
-        log_printer.info('Deleting old backup file... '
-                         + os.path.relpath(ofile))
+        logging.info('Deleting old backup file... ' + os.path.relpath(ofile))
         try:
             os.remove(ofile)
         except OSError as oserror:
             not_deleted += 1
-            log_printer.warn("Couldn't delete {}. {}".format(
+            logging.warning("Couldn't delete {}. {}".format(
                 os.path.relpath(ofile), oserror.strerror))
 
     if not_deleted:
-        log_printer.warn(str(not_deleted) + ' .orig backup files could not be'
-                         ' deleted, possibly because you lack the permission'
-                         ' to do so. coala may not be able to create'
-                         ' backup files when patches are applied.')
+        logging.warning(str(not_deleted) + ' .orig backup files could not be'
+                                           ' deleted, possibly because you '
+                                           'lack the permission to do so. '
+                                           'coala may not be able to create'
+                                           ' backup files when patches are '
+                                           'applied.')
```

Comment:
we should maybe add a commit on top using `str.format` 😅 
-->
```
logging.warning(
    '{} .orig backup files could not be ......'.format(not_deleted))
```
or add an issue afterwards :D